planner/core/logical_plan_test.go

```diff
@@ -1310,6 +1310,7 @@ func (s *testPlanSuite) TestVisitInfo(c *C) {
 		c.Assert(err, IsNil, comment)
 		// to fix, Table 'test.ttt' doesn't exist
 		err = Preprocess(s.ctx, stmt, WithPreprocessorReturn(&PreprocessorReturn{InfoSchema: s.is}))
+		c.Assert(err, IsNil, comment)
```

Comment:
This `err` seems to need a fix rather than simply think it's `nil` here. (see the comment)